src/mono/mono/metadata/assembly.c

```diff
@@ -5353,10 +5353,10 @@ mono_create_new_bundled_satellite_assembly (const char *name, const char *cultur
 }
 
 /**
- * mono_register_bundled_satellite_assemblies:
+ * mono_register_bundled_satellite_assemblies_internal:
```

Comment:
I don't like this name change. The function isn't even internal—we're exporting it!

Reply:
Ah right. It should be fine without the internal suffix as it has a different signature than the binding.

Reply:
Isn't one just `mono_register_bundled_satellite_assemblies` and the other `mono_wasm_register_bundled_satellite_assemblies` anyway?

Reply:
Yeah. That's what I meant 😃